tasks/pre_checks/001_validate_network_interfaces.yml

```diff
@@ -65,11 +65,44 @@
   - debug: var=bb_filtered_list
   - name: Generate output list
     set_fact:
-      otopi_host_net: >-
+      host_net: >-
         {{ [bridge_interface] if bridge_interface is defined else bb_filtered_list.results |
         reject('skipped') | map(attribute='bond_item.ansible_facts.otopi_net_host') | list }}
+    register: host_net
+  - debug: var=host_net
+  - name: Collect interface types
+    shell: nmcli con show {{ nic }} | grep connection.type
+    with_items:
+      - "{{ host_net }}"
+    loop_control:
+      loop_var: nic
+    register: interface_types
+  - debug: var=interface_types
+  - name: Check for Team devices
+    set_fact:
+      is_team: "{{ nic_if.stdout.find('team') > 0 }}"
+    when: nic_if.stdout.find('team') != -1
+    with_items:
+      - "{{ interface_types.results }}"
+    loop_control:
+      loop_var: nic_if
+    register: team_list
+  - debug: var=team_list
+  - name: Get list of team devices
+    set_fact:
+      team_if: "{{  team_list.results | reject('skipped') | map(attribute='nic_if.nic') | list }}"
+  - debug: var=team_if
+  - name: Filter unsupported interface types
+    set_fact:
+      otopi_host_net: "{{ host_net | difference(team_if) }}"
     register: otopi_host_net
   - debug: var=otopi_host_net
+  - name: Failed if only teaming devices are availible
+    fail:
+      msg: >-
+        Only Team devices {{ team_if | join(', ') }} are present.
```

Comment:
Change the Error message to : 
ERROR: Only Team devices are present: {{ team_if | join(', ') }}. Teaming is not supported.

I'm not familiar with the phrase : `networking teaming` please don't use it

Reply:
This phrase has been taken from RHV documentation:
https://access.redhat.com/documentation/en-us/red_hat_virtualization/4.3/html/planning_and_prerequisites_guide/recommendations#networking-recommendations
This is a fail message so it already contains 'ERROR' in the installation and log file